pkgs/by-name/vk/vkd3d-proton/package.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, fetchFromGitHub
+, glslang
+, meson
+, ninja
+, stdenv
+, wine
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "vkd3d-proton";
+  version = "2.11.1-unstable-2024-03-07";
+
+  src = fetchFromGitHub {
+    owner = "HansKristian-Work";
+    repo = "vkd3d-proton";
+    rev = "ffb04fb7091f932ca95da236d63ab9a4064d6443";
+    fetchSubmodules = true;
```

Comment:
Is there a point in devendoring things?

Reply:
Well, it can possibly work. I will try.

Reply:
Splitted at https://github.com/NixOS/nixpkgs/pull/319877